### PR TITLE
A22 AB2 13 - Add Warehouse

### PR DESCRIPTION
**Pull request by SomeViet**
Requesting merge to incorporate Add Warehouse page to development branch.

Post API functionality still in progress.

### CODE REVIEW

File: src/components/AddNewWarehouse/AddNewWarehouse.js [hunk 1]
```diff
@@ -1,8 +1,221 @@
...
+            },
+            errors: {},
+        };
+    }
+
+    contactSubmit(e) {
+        e.preventDefault();
+
+        if (this.handleValidation()) {
+            console.log("Form submitted");
```
Comment (ankurbag) [inline src/components/AddNewWarehouse/AddNewWarehouse.js:35]: Please remove the `console.logs`

File: src/components/AddNewWarehouse/AddNewWarehouse.js [hunk 2]
```diff
@@ -1,8 +1,221 @@
...
+            // add API
+            // alert("Form submitted");
+            // const port = 8080;
+            // axios.post(`http://localhost:${port}`, { var: "test" }).then().catch();
+        } else {
+            console.log("Form has errors.");
```
Comment (ankurbag) [inline src/components/AddNewWarehouse/AddNewWarehouse.js:41]: Please remove the `console.logs`

File: src/components/AddNewWarehouse/AddNewWarehouse.js [hunk 3]
```diff
@@ -1,8 +1,221 @@
+import "./AddNewWarehouse.scss";
 import React from "react";
+import { Link } from "react-router-dom";
+import {
+    BlueButton,
+    WhiteButton,
+    WarehouseInput,
+    AddEditSubheader,
+} from "../../components/index";
+// import axios from "axios";
```
Comment (ankurbag) [inline src/components/AddNewWarehouse/AddNewWarehouse.js:10]: I was wondering if we can add comments in this fomat  for intentional commented code:

```
  // TODO : Leaving the commented code once API is ready
  // import axios from "axios";
```

File: src/components/AddNewWarehouse/AddNewWarehouse.js [hunk 4]
```diff
@@ -1,8 +1,221 @@
...
+        };
+    }
+
+    contactSubmit(e) {
+        e.preventDefault();
+
+        if (this.handleValidation()) {
+            console.log("Form submitted");
+            // add API
+            // alert("Form submitted");
```
Comment (ankurbag) [inline src/components/AddNewWarehouse/AddNewWarehouse.js:37]: I was wondering if we can add comments in this fomat for intentional commented code:

  // TODO : Leaving the commented code once API is ready
  // all your commented code

File: src/components/index.js [hunk 5]
```diff
@@ -12,4 +12,9 @@ export { default as DeleteItem } from "./DeleteItem/DeleteItem";
 
 export { default as Footer } from "./Footer/Footer";
 
+export { default as BlueButton } from "./Buttons/BlueButton/BlueButton";
+export { default as WhiteButton } from "./Buttons/WhiteButton/WhiteButton";
+export { default as RedButton } from "./Buttons/RedButton/RedButton";
+
```
Comment (ankurbag) [inline src/components/index.js:18]: nit-pick : extra line here!

**Comment by ankurbag** [comment]
LGTM :) Please take care of minor comments. Approving it!

**Review by ankurbag — APPROVED** [review]

File: src/components/AddNewWarehouse/AddNewWarehouse.js (hunk 4 above)
Comment (SomeViet) [reply src/components/AddNewWarehouse/AddNewWarehouse.js:37]: Comment added. Done.

File: src/components/AddNewWarehouse/AddNewWarehouse.js (hunk 3 above)
Comment (SomeViet) [reply src/components/AddNewWarehouse/AddNewWarehouse.js:10]: Comment added. Done.

File: src/components/AddNewWarehouse/AddNewWarehouse.js (hunk 2 above)
Comment (SomeViet) [reply src/components/AddNewWarehouse/AddNewWarehouse.js:41]: Removed "else"

File: src/components/AddNewWarehouse/AddNewWarehouse.js (hunk 1 above)
Comment (SomeViet) [reply src/components/AddNewWarehouse/AddNewWarehouse.js:35]: Removed.

**Pull request merged**